vault/templates/rolebinding-eso.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.esoInit.enabled }}
```

Comment:
The same question as before about jobs. I think we need a single template for rolebinding